actor/src/main/scala/org/apache/pekko/dispatch/VirtualThreadSupport.scala

```diff
@@ -67,7 +68,7 @@ private[dispatch] object VirtualThreadSupport {
         MethodType.methodType(ofVirtualClass, classOf[String], classOf[Long]))
       // TODO support replace scheduler when we drop Java 8 support
       val factoryMethod = lookup.findVirtual(builderClass, "factory", MethodType.methodType(classOf[ThreadFactory]))
-      builder = nameMethod.invoke(builder, prefix + "-virtual-thread-", 0L)
+      builder = nameMethod.invoke(builder, prefix + "-virtual-thread-", zero)
```

Comment:
I find it hard to believe the boxing would be a bottleneck/hotspot in the creation of thread factories, but OK I suppose.

Reply:
Actually,  I was wondering should we have a thread I'd for virtual thread....